lib/olark/index.js

```diff
@@ -55,6 +68,35 @@ Olark.prototype.loaded = function(){
 };
 
 /**
+ * Listen for events.
+ */
+
+Olark.prototype.attachListeners = function(){
+  var self = this;
+
+  api('chat.onBeginConversation', function(){
+    self.analytics.track('Live Chat Conversation Started',
+      {},
+      { context: { integration: integration }}
+    );
+  });
+
+  api('chat.onMessageToOperator', function(event){
+    self.analytics.track('Live Chat Message Sent',
```

Comment:
Can you kick the first argument (`'Live Chat Message Sent'`) here to a new line for consistency/readability? (That way each argument gets its own newline.)